src/controller/handler/router.go

```diff
@@ -73,6 +76,9 @@ var routes = []Route{
     Route{"GET",   "/api/v1/repos/{namespace}/{name}",            wrapper(authWrapper(globalRepoWrapper(GetGlobalRepo)))},
     Route{"GET",   "/api/v1/repos/{namespace}/{name}/tags",       wrapper(authWrapper(globalRepoWrapper(ListGlobalRepoTag)))},
     Route{"GET",   "/api/v1/repos/{namespace}/{name}/tags/{tag}", wrapper(authWrapper(globalRepoWrapper(GetGlobalRepoTag)))},
+
+    // agent
+    Route{"GET", "/api/v1/agent/{user_key}", wrapperWS(ConnectAgent)},
```

Comment:
change to /api/v1/agent, add a func makeConnId(userKey, nodeUuid); see also https://github.com/Hyper2Stack/MoonLegend/pull/7